server/createApp.js

```diff
@@ -4,136 +4,115 @@ var methodOverride = require('method-override');
 var morgan = require('morgan');
 
 var language = require('../lib/middleware/language');
+var createComponentRouter = require('./createComponentRouter');
 
-module.exports = function createApp (keystone, express) {
+module.exports = function initRouter (router, routesFunc) {
 
-	if (!keystone.app) {
-		if (!express) {
-			express = require('express');
-		}
-		keystone.app = express();
-	}
-
-	var app = keystone.app;
-
-	keystone.initDatabase();
-	keystone.initExpressSession(keystone.mongoose);
+	var keystone = this;
```

Comment:
I don't understand why you assign `this` to `keystone` and then rename `keystone` almost everywhere to `this`?